app/assets/stylesheets/application.css.scss

```diff
@@ -53,6 +53,16 @@ body {
   color: #fff;
 }
 
+.link-to-text-color:hover {
+  color: #fff;
+}
+
+.home {
+  border: thin;
+  border-style: solid;
+  border-color: #461B7E;
```

Comment:
Color literals like `#461B7E` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#461B7E` should be written as `#461b7e`